arbnode/batch_poster.go

```diff
@@ -65,6 +67,7 @@ var DefaultBatchPosterConfig = BatchPosterConfig{
 	PostingErrorDelay:    time.Second * 10,
 	MaxBatchPostInterval: time.Hour,
 	CompressionLevel:     brotli.DefaultCompression,
+	DASRetentionPeriod:   time.Hour * 24 * 14,
```

Comment:
Let's make this 15 days.  (That way, the inbox logic can comfortably require the remaining lifetime to be >= 14 days.)